Clearify independend Reolink push notification switch (#30123)
home-assistant/home-assistant.github.io: source/_integrations/reolink.markdown

For NVRs, a global switch for **Record**, **Push**, **Buzzer**, **Email**, and **FTP** will be available under the NVR device as well as a switch per channel of the NVR under the camera device. The respective feature will only be active for a given channel if both the global and that channel switch are enabled (as is also the case in the Reolink app/client).

> **Push** notifications to a phone will only be provided if both the **Push notifications** switch in Home Assistant is ON (for NVRs both the global and channel switch) and the Push-notification in the Reolink App of that phone is ON. The Push-notification in the reolink app is independet of the Home Assistant setting and independent of the settings on other phones connected to the same camera. Reolink does this so you have a independed way of turning off push notifications per phone.

Reviewer: ```suggestion
**Push** notifications to a phone will only be provided if the following conditions are met: 
- The **Push notifications** switch in Home Assistant is ON. 
- For NVRs,  both the global and channel switch are ON.
- The Push-notification in the Reolink App of that phone is ON.

The Push-notification in the Reolink app is independent of the Home Assistant setting. It is also independent of the settings on other phones connected to the same camera. Reolink does this so you have an independent way of turning off push notifications per phone.
```
